src/build/Config.zig

```diff
@@ -306,6 +308,21 @@ pub fn init(b: *std.Build) !Config {
         break :emit_docs path != null;
     };
 
+    config.emit_terminfo = b.option(
+        bool,
+        "emit-terminfo",
+        "Install Ghostty terminfo source file",
+    ) orelse switch (target.result.os.tag) {
+        .windows => true,
+        else => false,
```

Comment:
I think this and termcap both should emit for Debug builds. It could be useful for dev builds.

Reply:
Done.